README.md

```diff
@@ -1,5 +1,11 @@
-# Fucker bot
+# Pull party bot
 
 [![Build Status](https://travis-ci.com/pool-party/fucker-bot.svg?branch=master)](https://travis-ci.com/pool-party/fucker-bot) [![codecov](https://codecov.io/gh/pool-party/fucker-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/pool-party/fucker-bot) [![CodeFactor](https://www.codefactor.io/repository/github/pool-party/fucker-bot/badge)](https://www.codefactor.io/repository/github/pool-party/fucker-bot)
 
-telegram bot notifier
\ No newline at end of file
+For the first time to setup required version of gradle and dependencies run:
+```./gradlew clean build``` for UNIX systems
+```gradlew clean build```   for windows
```

Comment:
`./gradlew.bat` is for windows, isn't it?

Reply:
> `./gradlew.bat` is for windows, isn't it?

Nope